packages/react-devtools-shared/src/__tests__/TimelineProfiler-test.js

```diff
@@ -1243,10 +1243,9 @@ describe('Timeline profiler', () => {
         function Example() {
           const setHigh = React.useState(0)[1];
           const setLow = React.useState(0)[1];
-          const startTransition = React.useTransition()[1];
 
           updaterFn = () => {
-            startTransition(() => {
+            React.startTransition(() => {
```

Comment:
This test needed to be updated because the lane changed from Continuous (which we're merging with Sync soon regardless) to Sync but since it doesn't really test anything specific to the hook form of `startTransition` I switched it to use the non-hook form instead, which doesn't perform a pending update. That way the test works when running against older versions of React, too.